x-pack/plugins/infra/public/pages/metrics/inventory_view/components/survey_section.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { FeatureFeedbackButton } from '../../../../components/feature_feedback_button';
+
+import { useWaffleOptionsContext } from '../hooks/use_waffle_options';
+import { SurveyKubernetes } from './survey_kubernetes';
+
+const INVENTORY_FEEDBACK_LINK = 'https://ela.st/survey-infra-inventory?usp=pp_url';
+
+export const SurveySection = () => {
+  const { nodeType } = useWaffleOptionsContext();
+  const podNodeType: typeof nodeType = 'pod';
+
+  return (
+    <>
+      {nodeType === podNodeType ? (
```

Comment:
`nodeType` is already typed as `nodeType`, we can let TS infer the union type.

```suggestion

  return (
    <>
      {nodeType === 'pod' ? (
```

Reply:
True, changed 👍